client/src/pages/private/ParticipantEOI.js

```diff
@@ -145,8 +165,28 @@ export default () => {
     history.push(pathToPush);
   };
 
+  const onConfirmInterest = async () => {
+    setOpenConfirmInterest(false);
+    setLoading(true);
+    const success = await submitConfirmInterestRequest(id);
+    if (success) {
+      openToast({
+        status: ToastStatus.Success,
+        message: 'Confirm interest success',
+      });
+      setDisableConfirmInterest(true);
+    } else {
+      openToast({
+        status: ToastStatus.Error,
+        message: 'Confirm interest failure',
```

Comment:
I'm adding getting toast message copy finalized to the prod readiness ticket FYI - so we don't need to worry about this for now